local-tests/test_multiple_restarts.py

```diff
@@ -33,49 +34,40 @@
 
 chain.set_flags_validator('validator')
 
-print('Starting the chain')
+printt('Starting the chain')
 chain.start('aleph')
 
-for run_duration, stop_duration, catch_up_duration in [[150, 20, 30], [20, 15, 30], [20, 15, 30]]:
-    print(f'Waiting {run_duration}s')
-    sleep(run_duration)
-
-    finalized_before_kill_per_node = check_finalized(chain)
-    print('Killing one validator and one nonvalidator')
+printt('Waiting for finalization')
+chain.wait_for_finalization(0)
+printt('Waiting for authorities')
+chain.wait_for_authorities()
 
+for sleep_duration in [21, 37, 15]:
```

Comment:
okay, I get papal numbers, but I'm afraid that the experiment will not fit into a single session as `README.md` claims (at least with `short-session` feature)

Reply:
This test is not meant to be run with `short-session` (see line 13)